Tests/varLib/instancer/instancer_test.py

```diff
@@ -1975,7 +2000,7 @@ def test_negative_var(self, var, axisTag, newMin, expected):
 def test_limitFeatureVariationConditionRange(oldRange, newLimit, expected):
     condition = featureVars.buildConditionTable(0, *oldRange)
 
-    result = instancer._limitFeatureVariationConditionRange(
+    result = instancer_featureVars._limitFeatureVariationConditionRange(
```

Comment:
why not just call `instancer.featureVars._limitFeatureVariationConditionRange`? why use the alias `instancer_featureVars` with the underscore